app/page/account/create/index.tsx

```diff
@@ -32,15 +32,18 @@ export function AccountCreate() {
   const createNewAccount = async () => {
     setAccountCreating(true)
     try {
-      if (!network.accountFactory[AccountType.PasskeyAccount]) {
+      if (
+        !network.accountFactory[AccountType.PasskeyAccount] ||
+        network.accountFactory[AccountType.PasskeyAccount] === "0x"
```

Comment:
I think we can extract this check as a function in `address` util (e.g., `address.isValid(...)`)